evalml/demos/fraud.py

```diff
@@ -23,8 +25,9 @@ def load_fraud(n_rows=None, verbose=True, return_pandas=False):
                      target="fraud",
                      n_rows=n_rows,
                      verbose=verbose)
-    X = X.set_types({"provider": "Categorical", "region": "Categorical"})
-
     if return_pandas:
-        return X.to_dataframe(), y.to_series()
+        return X, y
+
+    X.ww.init()
+    y = ww.init_series(y)
```

Comment:
My understanding is that `load_data` used to return WW types, so we didn't need to do the conversion here; are we updating `load_data` to return WW types in a separate PR?